.github/workflows/contracts-cd.qa.yml

```
@@ -0,0 +1,32 @@
+---
+name: Deploy Contracts CD for QA
+
+on:
+  push:
+    branches: [main, PP-675/contracts-cd]
```

We need to remove the test branch
```suggestion
    branches: [main]
```